filebeat/prospector/prospector_log.go

```diff
@@ -118,11 +122,14 @@ func (l *Log) getFiles() map[string]os.FileInfo {
 
 	paths := map[string]os.FileInfo{}
 
-	for _, glob := range l.config.Paths {
-		// Evaluate the path as a wildcards/shell glob
-		matches, err := filepath.Glob(glob)
+	for _, path := range l.config.Paths {
+		depth := uint8(0)
+		if l.config.recursiveGlob.enabled {
```

Comment:
I expected to see here `recursiveGlob.Enabled()` as this come with ucfg for each "object". Problem is that if we want that we would have to do unpack each time the scan runs which is not a good idea. So this looks good to me.